src/main/java/com/redhat/devtools/lsp4ij/settings/LanguageServerView.java

```diff
@@ -417,4 +418,22 @@ public List<ServerMappingSettings> getMappings() {
     public boolean isEditingCommand() {
         return languageServerPanel.getCommandLine() != null && languageServerPanel.getCommandLine().hasFocus();
     }
+
+    /**
+     * Tells whether the save tip balloon has been shown for this language server view
+     *
+     * @return true if save tip has been shown, else false
+     */
+    public boolean isSaveTipShown() {

```

Comment:
I don't think the save type shown must belong to the language server view which is the UI used inside LSP console.

My idea was to have a new entry in the standard IJ preferences 

 * Language Servers (existing preferences)
   * General (which display a checkbox to re-enable the tip.

But this General preferences is not important I think for this PR. The main important thing is to have a `Got It` options to disable this tip.

In otherwise, please remove this saveTipShown getter/setter in the LanguageServerView and after that I think PR should be ready.

Reply:
Yep, you're correct. Added a new field to ApplyLanguageServerSettingsAction to handle showing the balloon only once per configuration change and removed it from the LanguageServerView